File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
    Scanner myObj = new Scanner(System.in); 
    
    int a = myObj.nextInt();
    int b = myObj.nextInt();
    int c = myObj.nextInt();
    if(a + b + c == 180){
    if(a==b && b==c && c==a){
        System.out.println("Equilateral");
    }else if(a==b && b != c || b==c && c!=a || c==a && a != b){
        System.out.println("Isosceles");
    }else{
        System.out.println("Scalene");
    }
    }else{
        System.out.println("Error");
    }
    }
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
a = int(input())
b = int(input())
c = int(input())
  
if a + b + c == 180:
    if a==b and b==c and c==a:
        print("Equilateral");
    elif (a==b and b != c) or ( b==c and c!=a) or (c==a and a != b):
        print("Isosceles");
    else:
        print("Scalene");
else:
    print("Error");